src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -623,6 +624,17 @@ class KotlinCompilationTests {
 		assertClassLoadable(result, "${KotlinTestProcessor.GENERATED_PACKAGE}.${KotlinTestProcessor.GENERATED_JAVA_CLASS_NAME}")
 	}
 
+	@Test
+	fun `checks configuration with pluginClasspaths`() {
```

Comment:
Test name should mention the expected outcome.

Reply:
Fixed!